tools/bug-report/pkg/common/common.go

```diff
@@ -52,6 +53,16 @@ var (
 				"debug/push_status",
 				"debug/inject",
 			},
+			proxyDebugURLs: []string{
+				"certs",
+				"clusters",
+				"config_dump",
```

Comment:
can you add ` ?include_eds` to config_dump?

We do need to check that this param does not break  older proxies. @lambdai I have checked that  unknown params are ignored experimentally.

Reply:
anecdotally, In my experience it is ignored,

Reply:
Since istio-proxy 1.7 supports this param we should include it and cherry-pick it back to 1.7.